src/dialogs/vkbasalt.py

```diff
@@ -0,0 +1,326 @@
+            if self.expander_dls.get_enable_expansion() is True:
+                effects.append("dls")
+                vkbasalt_settings.dls_sharpness = Gtk.Adjustment.get_value(self.spin_dls_sharpness)
+                vkbasalt_settings.dls_denoise = Gtk.Adjustment.get_value(self.spin_dls_denoise)
+            if self.expander_fxaa.get_enable_expansion() is True:
+                effects.append("fxaa")
+                vkbasalt_settings.fxaa_subpixel_quality = Gtk.Adjustment.get_value(self.spin_fxaa_subpixel_quality)
+                vkbasalt_settings.fxaa_quality_edge_threshold = Gtk.Adjustment.get_value(self.spin_fxaa_quality_edge_threshold)
+                vkbasalt_settings.fxaa_quality_edge_threshold_min = Gtk.Adjustment.get_value(self.spin_fxaa_quality_edge_threshold_min)
+            if self.expander_smaa.get_enable_expansion() is True:
+                effects.append("smaa")
+                vkbasalt_settings.smaa_threshold = Gtk.Adjustment.get_value(self.spin_smaa_threshold)
+                vkbasalt_settings.smaa_edge_detection = self.smaa_edge_detection
+                vkbasalt_settings.smaa_corner_rounding = Gtk.Adjustment.get_value(self.spin_smaa_corner_rounding)
+                vkbasalt_settings.smaa_max_search_steps = Gtk.Adjustment.get_value(self.spin_smaa_max_search_steps)
+                vkbasalt_settings.smaa_max_search_steps_diagonal = Gtk.Adjustment.get_value(self.spin_smaa_max_search_steps_diagonal)
+
+            vkbasalt_settings.effects = tuple(effects)
+
+        else:
+            vkbasalt_settings.effects = False
+
+        if self.btn_lut_file_path:
+            vkbasalt_settings.lut_file_path = self.btn_lut_file_path
+
+        vkbasalt_settings.output = conf
+
+        parse(vkbasalt_settings)
+        self.close()
+        return GLib.SOURCE_REMOVE
+
+    def __save(self, *args):
+        GLib.idle_add(self.__idle_save)
+
+    def __check_state(self, *args):
+        if True in [
+            self.expander_cas.get_enable_expansion(),
+            self.expander_dls.get_enable_expansion(),
+            self.expander_fxaa.get_enable_expansion(),
+            self.expander_smaa.get_enable_expansion(),
+        ] or self.btn_lut_file_path is not False:
+            self.btn_save.set_sensitive(True)
+        else:
+            self.btn_save.set_sensitive(False)
+
+    def __default(self, widget, state):
+        self.expander_cas.set_sensitive(not state)
+        self.expander_dls.set_sensitive(not state)
+        self.expander_fxaa.set_sensitive(not state)
+        self.expander_smaa.set_sensitive(not state)
+        self.action_clut.set_sensitive(not state)
+        if state is False:
+            if self.expander_cas.get_enable_expansion() is False and self.expander_dls.get_enable_expansion() is False and self.expander_fxaa.get_enable_expansion() is False and self.expander_smaa.get_enable_expansion() is False and self.btn_lut_file_path is False:
```

Comment:
consider truncating long lines, PEP8 suggest 72 chars (we don't follow it strictly but this line is very long)
https://peps.python.org/pep-0008/#maximum-line-length

e.g. using `\`
```python
if self.expander_cas.get_enable_expansion() is False \
    and self.expander_dls.get_enable_expansion() is False \
    and self.expander_fxaa.get_enable_expansion() is False \
    and self.expander_smaa.get_enable_expansion() is False \
    and self.btn_lut_file_path is False:
```

Reply:
Or 

```python
if (
    x
    and y
    or z
):
```

which is the PEPXYZ way to doing it and what `black` would do